types/token.go

```diff
@@ -75,7 +75,7 @@ const (
 	STELLAR      TokenType = "STELLAR"
 	KRC20        TokenType = "KRC20"
 	AURORA       TokenType = "AURORA"
-	ALGORAND     TokenType = "ALGORAND"
```

Comment:
Why are we changing this? Can you provide the references

Reply:
@iCell type should be ASA not ALGORAND https://github.com/trustwallet/assets/pull/25956#discussion_r1283776125

We have 1 asset in Cardano folder and it has ALGORAND type, I noticed what new assets Nazar trying to add have ASA type. They said all assets should have ASA type

Reply:
Thanks for clarifying, please create an issue to updating the type in assets database, also preparing the SQL for it.

No rush, can be done next Monday

Reply:
SQL executed